### PR TITLE
extend coverage with new unittests

### PR DESCRIPTION
**Pull request by apiddubny**
## Description
- increase unittest coverage up to 95%
- change tests folder structer
- add unittests for OpenStackShellModelParser class
- add unittests for NovaInstanceService class

## Breaking
NO

---
This change is [Reviewable](https://reviewable.io/reviews/qualisystems/openstack-shell/29)

### CODE REVIEW

**Comment by alexazarh** [comment]
@gabhijit-izel Please review this

File: package/tests/test_cp/test_openstack/test_common/test_deploy_data_holder.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+from cloudshell.cp.openstack.common.deploy_data_holder import DeployDataHolder
+
+
+class TestDeployDataHolder(TestCase):
+
+    def setUp(self):
+        super(TestDeployDataHolder, self).setUp()
+        self.tested_class = DeployDataHolder
+
+    def test_create_from_params(self):
```
Comment (gabhijit-izel) [inline package/tests/test_cp/test_openstack/test_common/test_deploy_data_holder.py]: This is not the problem with the way test-case is written per se, but I believe `create_from_params` is a side effect of original code. So I will accept this change, but later on delete, both the original code and this test case.

Comment (apiddubny) [inline package/tests/test_cp/test_openstack/test_common/test_deploy_data_holder.py]: ok

**Comment by gabhijit-izel** [comment]
Reviewed 27 of 27 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/qualisystems/openstack-shell/29)*

**Comment by gabhijit-izel** [comment]
@apiddubny : My understanding of unitTesting is not good. I wanted to ask why you used mock.MagicMock() in some unit tests.

**Pull request merged**